config/routes.rb

```diff
@@ -84,6 +84,8 @@ def matches?(_)
   get '/zh', to: 'home#index'
   get '/tl', to: 'home#index'
 
+  get '/sign-in', to: 'auth#sign_in'
```

Comment:
so you'll have to add /en/es/zh/tl variations here. If you have ideas on how to simplify it as a regex let me know!

Reply:
can't we add url param? like `/?lang=en`

Reply:
No, we want it to be interoperable with angular, and angular adds the language prefix to the path.

It's also the way bloom does it, so the workarounds wouldn't be worth it.